guides/common/modules/proc_running-salt.adoc

```diff
@@ -1,43 +1,39 @@
 [id="Running_Salt_{context}"]
 = Running Salt
 
-You can run arbitrary Salt functions using remote execution.
-
-Most commonly, you can run `salt.highstate` on one or more Salt Minions.
+You can run arbitrary Salt functions using remote execution, for example `salt.highstate`, on one or more Salt Minions.
 This applies all relevant Salt States on your managed hosts.
 
-.Procedure
-. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
-. Select one or multiple hosts.
-. Click *Run Salt* from the _actions_ drop down menu.
+.Running Salt state.highstate
+. In the {ProjectWebUI}, navigate to *Monitor* > *Jobs* and click *Run job*.
+. Select *Salt* as *Job category* and *Salt Run state.highstate {endash} Salt default* as *Job template* and click *Next*.
```

Comment:
Would it work better to keep it as a single procedure, but just to branch this step, when the rest of the procedure is the same?

Reply:
Good call; merged into one.